http/src/request/guild/ban/get_bans.rs

```diff
@@ -7,37 +7,105 @@ use crate::{
 };
 use twilight_model::{
     guild::Ban,
-    id::{marker::GuildMarker, Id},
+    id::{
+        marker::{GuildMarker, UserMarker},
+        Id,
+    },
 };
+use twilight_validate::request::{
+    get_guild_bans_limit as validate_get_guild_bans_limit, ValidationError,
+};
+
+struct GetBansFields {
+    after: Option<Id<UserMarker>>,
+    before: Option<Id<UserMarker>>,
+    limit: Option<u16>,
+}
 
 /// Retrieve the bans for a guild.
 ///
 /// # Examples
 ///
-/// Retrieve the bans for guild `1`:
+/// Retrieve the first 25 bans of a guild after a particular user ID:
 ///
 /// ```no_run
+/// use std::env;
 /// use twilight_http::Client;
 /// use twilight_model::id::Id;
 ///
 /// # #[tokio::main]
 /// # async fn main() -> Result<(), Box<dyn std::error::Error>> {
-/// let client = Client::new("my token".to_owned());
+/// let client = Client::new(env::var("DISCORD_TOKEN")?);
 ///
 /// let guild_id = Id::new(1);
+/// let user_id = Id::new(2);
 ///
-/// let bans = client.bans(guild_id).exec().await?;
+/// let response = client.bans(guild_id).after(user_id).exec().await?;
```

Comment:
The default is `1000` so this needs to be set to `25` explicitly.
```suggestion
/// let response = client.bans(guild_id).after(user_id).limit(25).exec().await?;
```